app/controllers/candidate_interface/degrees/base_controller.rb

```diff
@@ -27,5 +29,15 @@ def degrees_params
         :predicted_grade, :award_year
       )
     end
+
+    def render_new
+      degrees = DegreesForm.build_from_application(current_candidate.current_application)
```

Comment:
maybe 'render_new' should just render, instead of also `DegreesForm.build_from_application` ?

Reply:
It needs to create a `DegreesForm` in order to work out which view to render. Is your suggestion to for `render_new` to take in a `DegreesForm`?